mflowgen/tile_array/construct.py

```diff
@@ -131,7 +135,8 @@ def construct():
   lvs.extend_inputs( ['sram.spi'] )
 
   # Extra dc inputs
-  dc.extend_inputs( dc_postcompile.all_outputs() )
+  #dc.extend_inputs( dc_postcompile.all_outputs() )
+  synth.extend_inputs( dc_postcompile.all_outputs() )
```

Comment:
Should this still be here? Don't think we're using dc_postcompile anymore with the new genus node.